src/pipecat/processors/frameworks/rtvi.py

```diff
@@ -319,7 +321,11 @@ async def handle_function_call(
         message = RTVILLMFunctionCallMessage(data=fn)
         await self._push_transport_message(message, exclude_none=False)
 
-    async def handle_function_call_start(self, function_name: str):
+    async def handle_function_call_start(
+            self,
+            llm: FrameProcessor,
+            context: OpenAILLMContext,
+            function_name: str):
```

Comment:
nit: should we move `function_name` to be the first argument after `self`, so that the order is the same as for `handle_function_call()`?

Reply:
sounds good!